### PR TITLE
35 - Drupal 10 readiness

### LINKED ISSUE #35: Drupal 10 readiness
Update the packages and the related codebase to be Drupal 10 ready.

### PR DESCRIPTION
**Pull request by balintpekker**
Issue #35 

@changed
- `"bex/behat-screenshot": "^1.2"` to `"drevops/behat-screenshot": "^1.2",`
- Updated packages to be D10 ready (along with dependencies such as Symfony 6)
- Used `post-drupal-scaffold-cmd` hook to create settings.php and services.yml **before** the prepare command, fixing the `File 'tests/fixtures/project_01/docroot/sites/default/settings.php' is missing` issue on `post-install-cmd`.
- Updated `Cheppers\DrupalExtension\Context\Drupal\CoreCkeditor` context for CKEditor5.
- Updated drupal configurations to match D10.
- Added contrib modules and themes (classy, bartik, seven)
- Removed unnecessary `escapeshellarg()` calls and updated `Symfony\Component\Process\Process` on drupal install to match Symfony 6 (first argument must be an array, or should use `Process::fromShellCommandline()` method.

### CODE REVIEW

**Comment by balintpekker** [comment]
@todo Update `Cheppers\DrupalExtension\Context\Drupal\CoreCkeditor::doFillInWysiwygOnFieldWith()` to be ready with CKEditor5. (1 test feature is failing -- Feature: Test steps in \Cheppers\DrupalExtension\Context\Drupal\CoreCkeditor)

File: src/Context/Drupal/CoreCkeditor.php [hunk 1]
```diff
@@ -40,9 +40,23 @@ protected function ckeditorSetData(string $fieldId, string $newValue)
...
         $newValueSafe = addslashes($newValue);
 
+        $editor = "div.js-form-item-$fieldIdSafe-0-value .ck-editor__editable";
+
         $this
             ->getSession()
-            ->executeScript("CKEDITOR.instances['{$fieldIdSafe}'].setData('{$newValueSafe}');");
+            ->executeScript("
+      var domEditableElement = document.querySelector(\"$editor\");
+      if (domEditableElement.CKEditorInstance) {
```
Comment (balintpekker) [inline src/Context/Drupal/CoreCkeditor.php]: Currently, this is causing the issue: `TypeError: Cannot read properties of null (reading 'CKEditorInstance')
          at <anonymous>:3:30 (Behat\Mink\Exception\DriverException)`.

Comment (balintpekker) [reply src/Context/Drupal/CoreCkeditor.php]: Fixed in commit [6b2c35b](https://github.com/Cheppers/drupal-behat-contexts/pull/34/commits/6b2c35b47735f84e63ca43a4a92aa01e8708456e).

File: composer.json [hunk 2]
```diff
@@ -169,17 +181,17 @@
...
-                "sites/default/default.services.yml": "[web-root]/sites/default/services.yml",
-                "sites/default/default.settings.php": "[web-root]/sites/default/settings.php"
             }
         }
     },
     "scripts": {
         "post-install-cmd": [
             "@prepare"
         ],
+        "post-drupal-scaffold-cmd": [
```
Comment (gdeni) [inline composer.json:191]: Please add `"cp tests/behat/behat.local.example.yml tests/behat/behat.local.yml",` to create local behat yml file.

File: composer.json [hunk 3]
```diff
@@ -169,17 +181,17 @@
...
+            "cp tests/fixtures/project_01/docroot/sites/default/default.settings.php tests/fixtures/project_01/docroot/sites/default/settings.php",
+            "cp tests/fixtures/project_01/docroot/sites/default/default.services.yml tests/fixtures/project_01/docroot/sites/default/services.yml"
+        ],
         "lint": [
             "@lint:phpcs"
```
Comment (gdeni) [inline composer.json:196]: Please run `composer run lint` and fix the warnings/errors.

**Review by gdeni — COMMENTED** [review]
- The commands in the readme have been tested and run successfully.
- New D10 package has been tested in a client project. Behat tests have been run successfully. Screenshots have been created successfully of failed scenarios.
- Code looks fine

**Review by gdeni — APPROVED** [review]

File: composer.json (hunk 3 above)
Comment (balintpekker) [reply composer.json:196]: Resolved in [2eab336](https://github.com/Cheppers/drupal-behat-contexts/pull/34/commits/2eab336a83f6e1432565af7dc9f658aef84d322e).

**Pull request merged**